File: C/TOMADA13.c
#include<stdio.h>

int main()
{
	int t=0;
	int i=0;
	int total=0;

	for(i=0;i<4;i++)
	{
		scanf("%d", &t);

		if(t>=2 && t<=6)
		{
			total = total + t;
		}
		else
		{
			return 0;
		}
	}
	total = total-3;
	printf("%d", total);
	return 0;
}

File: Java/TOMADA13.java
import java.util.Scanner;

public class TOMADA13 {

	public static void main(String[] args) {
		
		Scanner inputs = new Scanner(System.in);
		int electrical_outlets=0;
		int total =0;
		
		for(int i=0;i<4;i++)
		{
			electrical_outlets = inputs.nextInt();
			if(electrical_outlets>=2 && electrical_outlets<=6)
			{
				total = total + electrical_outlets;
			}else{
				System.exit(0);
			}
		}
		
		total = total - 3;
		System.out.println(total);
	}

}
